### ISSUE TITLE
same problem with "Problem with connection in new version #3771"

### ISSUE THREAD

**Issue by Aidomain** [ethereum/mist#3917]
* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

same problem with "Problem with connection in new version #3771"
trying to update the geth to 1.8.8 in Ethereum-Wallet-win32-0-10-0, same problem happen.
Even I run geth --fast manually, it still happen.
```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by Aidomain** [comment]
the following is the log content:
[2018-05-31T00:45:44.131] [INFO] main - Starting in Wallet mode
[2018-05-31T00:45:44.650] [INFO] Db - Loading db: C:\Users\Administrator\AppData\Roaming\Ethereum Wallet\mist.lokidb
[2018-05-31T00:45:44.676] [INFO] Windows - Creating commonly-used windows
[2018-05-31T00:45:44.677] [INFO] Windows - Create secondary window: loading, owner: notset
[2018-05-31T00:45:44.919] [INFO] updateChecker - Check for update...
[2018-05-31T00:45:49.371] [INFO] Windows - Create primary window: main, owner: notset
[2018-05-31T00:45:49.388] [INFO] Windows - Create primary window: splash, owner: notset
[2018-05-31T00:45:50.646] [INFO] (ui: popupWindow) - Web3 not yet initialized, doing so now with HttpProvider.
[2018-05-31T00:45:50.721] [INFO] (ui: popupWindow) - Meteor starting up...
[2018-05-31T00:45:50.836] [INFO] (ui: splashscreen) - Web3 already initialized, re-using provider.
[2018-05-31T00:45:50.901] [INFO] (ui: splashscreen) - Meteor starting up...
[2018-05-31T00:45:50.919] [INFO] (ui: popupWindow) - Web3 already initialized, re-using provider.
[2018-05-31T00:45:50.974] [INFO] (ui: popupWindow) - Meteor starting up...
[2018-05-31T00:45:51.825] [INFO] ClientBinaryManager - Initializing...
[2018-05-31T00:45:51.825] [INFO] ClientBinaryManager - Checking for new client binaries config from: https://raw.githubusercontent.com/ethereum/mist/master/clientBinaries.json
[2018-05-31T00:45:52.927] [INFO] ClientBinaryManager - No "skippedNodeVersion.json" found.
[2018-05-31T00:45:52.928] [INFO] ClientBinaryManager - Initializing...
[2018-05-31T00:45:52.929] [INFO] ClientBinaryManager - Resolving platform...
[2018-05-31T00:45:52.929] [INFO] ClientBinaryManager - Calculating possible clients...
[2018-05-31T00:45:52.930] [INFO] ClientBinaryManager - 1 possible clients.
[2018-05-31T00:45:52.931] [INFO] ClientBinaryManager - Verifying status of all 1 possible clients...
[2018-05-31T00:45:52.932] [INFO] ClientBinaryManager - Verify Geth status ...
[2018-05-31T00:45:53.979] [INFO] ClientBinaryManager - Checking for Geth sanity check ...
[2018-05-31T00:45:53.979] [INFO] ClientBinaryManager - Checking sanity for Geth ...
[2018-05-31T00:46:05.790] [INFO] Sockets/node-ipc - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2018-05-31T00:46:05.792] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:06.792] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:07.792] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:08.791] [ERROR] Sockets/node-ipc - Connection failed (3000ms elapsed)
[2018-05-31T00:46:08.792] [WARN] EthereumNode - Failed to connect to node. Maybe it's not running so let's start our own...
[2018-05-31T00:46:08.792] [INFO] EthereumNode - Node type: geth
[2018-05-31T00:46:08.792] [INFO] EthereumNode - Network: main
[2018-05-31T00:46:08.792] [INFO] EthereumNode - SyncMode: light
[2018-05-31T00:46:08.793] [INFO] EthereumNode - Start node: geth main light
[2018-05-31T00:46:08.794] [INFO] EthereumNode - Start node using C:\Users\Administrator\AppData\Roaming\Ethereum Wallet\binaries\Geth\unpacked\geth.exe
[2018-05-31T00:46:14.728] [INFO] EthereumNode - INFO [05-31|00:46:14] Maximum peer count                       ETH=0 LES=100 total=25

[2018-05-31T00:46:17.214] [INFO] EthereumNode - INFO [05-31|00:46:17] Starting peer-to-peer node               instance=Geth/v1.8.8-stable-2688dab4/windows-386/go1.10.1
INFO [05-31|00:46:17] Allocated cache and file handles         database=C:\\Users\\Administrator\\AppData\\Roaming\\Ethereum\\geth\\lightchaindata cache=384 handles=1024

[2018-05-31T00:46:17.595] [INFO] EthereumNode - INFO [05-31|00:46:17] Initialised chain configuration          config="{ChainID: 1 Homestead: 1150000 DAO: 1920000 DAOSupport: true EIP150: 2463000 EIP155: 2675000 EIP158: 2675000 Byzantium: 4370000 Constantinople: <nil> Engine: ethash}"
INFO [05-31|00:46:17] Disk storage enabled for ethash caches   dir=C:\\Users\\Administrator\\AppData\\Roaming\\Ethereum\\geth\\ethash count=3
INFO [05-31|00:46:17] Disk storage enabled for ethash DAGs     dir=C:\\Users\\Administrator\\AppData\\Ethash                          count=2

[2018-05-31T00:46:17.598] [INFO] EthereumNode - INFO [05-31|00:46:17] Added trusted checkpoint                 chain=mainnet block=5603327 hash=3bb2c2…c9a356

[2018-05-31T00:46:17.599] [INFO] EthereumNode - INFO [05-31|00:46:17] Loaded most recent local header          number=0 hash=d4e567…cb8fa3 td=17179869184
INFO [05-31|00:46:17] Starting P2P networking 

[2018-05-31T00:46:17.729] [INFO] EthereumNode - 3000ms elapsed, assuming node started up successfully
[2018-05-31T00:46:17.729] [INFO] EthereumNode - Started node successfully: geth main light
[2018-05-31T00:46:17.732] [INFO] Sockets/node-ipc - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2018-05-31T00:46:17.733] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:18.734] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:19.735] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:20.736] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:21.737] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:22.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:23.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:24.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:25.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:26.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:27.738] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:28.739] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:29.739] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:30.626] [INFO] updateChecker - App is up-to-date.
[2018-05-31T00:46:30.739] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:31.739] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:32.739] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:33.740] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:34.740] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:35.741] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:36.741] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:37.743] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:38.744] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:39.745] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:40.746] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:41.747] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:42.748] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:43.748] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:44.748] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:45.748] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:46.749] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2018-05-31T00:46:47.734] [ERROR] Sockets/node-ipc - Connection failed (30000ms elapsed)
[2018-05-31T00:46:47.734] [ERROR] EthereumNode - Failed to connect to node Error: Unable to connect to socket: timeout
    at Timeout.setTimeout [as _onTimeout] (E:\2018\Ethereum-Wallet-win32-0-10-0\resources\app.asar\modules\sockets\base.js:93:27)
    at ontimeout (timers.js:469:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:264:5)
[2018-05-31T00:46:47.740] [ERROR] EthereumNode - Failed to start node Error: Unable to connect to socket: timeout
    at Timeout.setTimeout [as _onTimeout] (E:\2018\Ethereum-Wallet-win32-0-10-0\resources\app.asar\modules\sockets\base.js:93:27)
    at ontimeout (timers.js:469:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:264:5)

anyone can help?

**Comment by marcgarreau** [comment]
This may fall into a category of issues resolved with the [0.11.0 release](https://github.com/ethereum/mist/releases/tag/v0.11.0). Please give that a try and open if a new issue if appropriate.
